sdk/spring/azure-spring-boot/src/test/java/com/azure/spring/aad/webapi/validator/AADJwtIssuerValidatorTest.java

```diff
@@ -16,11 +19,15 @@
 
     final AADAuthenticationProperties aadAuthenticationProperties = mock(AADAuthenticationProperties.class);
     final Jwt jwt = mock(Jwt.class);
-
+    private AADTrustedIssuerRepository aadTrustedIssuerRepository = mock(AADTrustedIssuerRepository.class);
     @Test
     public void testIssuerSuccessVerify() {
+        List<String> trustedIssuers = new ArrayList<>();
+        trustedIssuers.add("https://sts.windows.net/fake-tenant-id/v1.0");
+        trustedIssuers.add("https://sts.windows.net/fake-tenant-id/v2.0");
         when(aadAuthenticationProperties.getTenantId()).thenReturn("fake-tenant-id");
         when(jwt.getClaim(AADTokenClaim.ISS)).thenReturn("https://sts.windows.net/fake-tenant-id/v2.0");
+        when(aadTrustedIssuerRepository.getTrustedIssuers()).thenReturn(trustedIssuers);
 
         AADJwtIssuerValidator validator = new AADJwtIssuerValidator();
```

Comment:
Seems like the AADJwtIssuerValidator  is not relevant to AADTrustedIssuerRepository ?